test/integration/testdata/egress-rule-httpbin.yaml.tmpl

```diff
@@ -0,0 +1,11 @@
+apiVersion: config.istio.io/v1alpha2
+kind: EgressRule
+metadata:
+  name: httpbin
+spec:
+  destination:
+      service: "httpbin.org"
```

Comment:
change this to nghttp2.org/httpbin/
and use http2

Reply:
hmm, http2 to nghttp2.org seems not to work. Let me debug it tomorrow and add nghttp2.org as an additional case, in a separate PR.